db/seeds.rb

```diff
@@ -32,7 +32,7 @@
 #   end
 # end
 
-current_csv_importers = ['Spawning Success','Tagged Animal Assessment','Untagged Animal Assessment']
+current_csv_importers = CsvImporter::CATEGORIES
```

Comment:
Part of the effort to consolidate the list of what categories are importable in one place and referencing that single declaration.